### DISCUSSION TITLE
"Clip info" Not Displayed

### DISCUSSION THREAD

**Discussion by bozackt** [smplayer-dev/smplayer#319] [Q&A] [answered]
Hi,

I have two smplayer installations on different Linux Mint 19.3 machines. Under "Information and properties...", one displays "Clip info" and one doesn't. As far as I can tell, both installations are identical. Both are running smplayer version 18.2.2.

Seeing the file metadata is important to me. How do I make "Clip info" display in both installations.

Thanks ...

**Comment by smplayer-dev** [comment, marked as answer]
The clip info comes from mpv / mplayer, so be sure you're using the same version of them too.

**Reply by bozackt** [reply]
Both computers have the same versions of mplayer (1.3.0) and mpv (0.27.2).

**Reply by smplayer-dev** [reply]
Be sure the "multimedia engine" (preferences -> general) is also the same in both machines.

**Reply by bozackt** [reply]
Thank you, Maintainer. That was the problem. It looks like the multimedia engine needs to be "mpv" for "Clip info" to be displayed. Previously, it was set to "mplayer".
